internal/plugin/plugin.go

```diff
@@ -37,7 +37,7 @@ type Plugin struct {
 }
 
 func (p *Plugin) String() string {
-	return fmt.Sprintf("%s:%s", p.Identity.IdentityString(), p.Version)
+	return fmt.Sprintf("%s:%s", p.Identity.IdentityString(), p.PluginVersion)
```

Comment:
Version here is `v1` of the config, it should be the plugin version.

Reply:
Doh - good catch!